torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1662,6 +1662,18 @@ int listClear(Stack& stack) {
   return 0;
 }
 
+template <typename T>
+int listDelete(Stack& stack) {
+  auto idx = pop(stack).toInt();
+  c10::List<T> list = pop(stack).to<c10::List<T>>();
+  if (idx > int64_t(list.size()) - 1) {
+    AT_ERROR("list assignment index out of range");
```

Comment:
nit: list deletion ?

Reply:
That does make sense but this is the same error Python puts out

Reply:
Ho interesting. Ok then.